src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -858,8 +858,9 @@ describe('manage nsfs cli bucket flow', () => {
             secret_key: 'G2AYaMpU3zRDcRFWmvzgQr9MoHIAsD+3oEXAMPLE',
         };
 
+        const bucket_name = 'bucket1';
```

Comment:
do you mind renaming this file to test_nc_bucket_cli.test.js (remove the nsfs) ?

Reply:
I don't mind, I will do it in a separate PR because we also have a test file for `test_nc_nsfs_account_cli.test.js` and they are also mentioned in the file `CI&Tests.md`.
I will add it to our list in the issue #8056 in the meantime.

Reply:
we don't need a new PR, I'm already renaming `test_nc_nsfs_account_cli.test.js`  in https://github.com/noobaa/noobaa-core/pull/8774/files